### PR TITLE
Add Framework guides

### PR DESCRIPTION
**Pull request by xinnks**
Add framework guides for:
- Next.js
- Nuxt
- Astro
- Qwik
- Remix
- Flask
- Tauri
- SvelteKit
- Flutter
- Quasar

The pages can be viewed but the links to them haven't been listed yet.

### CODE REVIEW

File: guides/frameworks/tauri.mdx [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
 
 ![Tauri banner](/images/guides/tauri-banner.png)
 
 ## Prerequisites
 
-To get the most out of this guide, you'll need to:
+Before you start, make sure you:
 
-- [Install the Turso CLI and create a database].
-- [Have a Tauri app].
```
Comment (notrab) [inline guides/frameworks/tauri.mdx]: @xinnks I'd prefer us to add the links inline.

File: guides/frameworks/tauri.mdx [hunk 2]
```diff
@@ -111,7 +113,7 @@ async fn get_all_items() -> Result<Vec<Item>> {
     let conn = db.connect()?;
 
     let mut results = conn
-        .query("SELECT * FROM items", ())
+        .query("SELECT * FROM table_name", ())
```
Comment (notrab) [inline guides/frameworks/tauri.mdx:116]: @xinnks I'm open to using a specific table name if you think it's makes more sense... or better... we use a real table name and include the example schema or SQL dump for people to create a db from if following... 

Maybe we do that in an update though. We don't need that for this initial release.

**Comment by notrab** [comment]
@xinnks let's get this rebased and then tomorrow we can merge 🚀 

Amazing work!

**Comment by xinnks** [comment]
@notrab Closing this in favour of https://github.com/tursodatabase/turso-docs/pull/130

**Pull request closed without merging**